x/vault/client/testutil/helpers.go

```diff
@@ -122,20 +108,42 @@ func (s *VaultIntegrationTestSuite) CreateNewAsset(name, denom string, price uin
 	}
 	s.Require().NotZero(assetID)
 
-	market := markettypes.Market{
-		Symbol:   name,
-		ScriptID: 12,
-		Rates:    price,
+	twa1 := markettypes.TimeWeightedAverage{
```

Comment:
shouldn't be this way, please refer fix from the liquidity module. 
this causes test case failure